index.html

```diff
@@ -170,6 +168,11 @@ <h2 id="project_tagline"></h2>
 <pre><code> $ java -jar publish-cli.jar -f publishMessages.json -en rabbit -mb mb123 -np -mp eiffelsemantics
 </code></pre>
 
+<p><strong>For loading protocol jars other than eiffelsemantics :</strong></p>
+
+<pre><code> $ java -Djava.ext.dirs="/usr/home/jars/" -jar publish-cli.jar -f publishMessages.json -mp protocolType
```

Comment:
Isn't it better to have an example path like "/path/to/jars" to make it obvious the user have to provide there own path?